src/main/resources/data-repository-openapi.yaml

```diff
@@ -267,20 +297,26 @@ paths:
       parameters:
       - $ref: '#/parameters/Id'
       responses:
-        202: # kicks off a delete study job
-          description: Study deletion job started & url for polling is in the response header
-          headers:
-            location:
-              type: string
-              description: url for the job poller
+        200:
```

Comment:
Are we just changing this to synchronous temporarily? If so, can we add a comment to that affect?

Reply:
As we discussed, we will keep study create and delete synchronous for now, so I removed the 202.